source/_docs/ecosystem/nginx.markdown

```diff
@@ -111,6 +111,7 @@ http {
 
         # These shouldn't need to be changed
         listen 443 default_server;
+        listen [::]:443 default_server ipv6only=on;
```

Comment:
Just tested and confirmed working, I don't feel strongly about saving a single line.
```
listen [::]:443 ssl default_server ipv6only=off;
```

We should however note that http2 can be enabled given a recent enough nginx version.